src/Aspire.Hosting.Azure.AppContainers/AzureContainerAppEnvironmentResource.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.Azure.AppContainers;
+
+/// <summary>
+/// 
+/// </summary>
+/// <param name="name"></param>
+/// <param name="configureInfrastructure"></param>
+public class AzureContainerAppEnvironmentResource(string name, Action<AzureResourceInfrastructure> configureInfrastructure) :
+    AzureProvisioningResource(name, configureInfrastructure)
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public BicepOutputReference ContainerAppEnvironmentId => new("AZURE_CONTAINER_APPS_ENVIRONMENT_ID", this);
+
+    /// <summary>
+    /// 
+    /// </summary>
+    public BicepOutputReference ContainerAppDomain => new("AZURE_CONTAINER_APPS_ENVIRONMENT_DEFAULT_DOMAIN", this);
+
+    /// <summary>
+    /// 
+    /// </summary>
+    public BicepOutputReference ManagedIdentityId => new("AZURE_CONTAINER_REGISTRY_MANAGED_IDENTITY_ID", this);
+
+    /// <summary>
+    /// 
+    /// </summary>
+    public BicepOutputReference ContainerRegistryUrl => new("AZURE_CONTAINER_REGISTRY_ENDPOINT", this);
+
+    /// <summary>
+    /// 
+    /// </summary>
+    public BicepOutputReference ContainerRegistryManagedIdentityId => new("AZURE_CONTAINER_REGISTRY_MANAGED_IDENTITY_ID", this);
+
+    /// <summary>
+    /// 
+    /// </summary>
+    public BicepOutputReference ClientId => new("MANAGED_IDENTITY_CLIENT_ID", this);
```

Comment:
Is it weird that "ClientId" and "ManagedIdentityId" point to different identities? Should ManagedIdentityId be renamed to include "Registry"?